src/app/components/user/UserConnectedAccount.js

```diff
@@ -0,0 +1,214 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Relay from 'react-relay/classic';
+import { FormattedMessage, defineMessages, injectIntl } from 'react-intl';
+import { ListItem } from 'material-ui/List';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import TextField from '@material-ui/core/TextField';
+import FASlack from 'react-icons/lib/fa/slack';
+import FAFacebook from 'react-icons/lib/fa/facebook-official';
+import FATwitter from 'react-icons/lib/fa/twitter';
+import rtlDetect from 'rtl-detect';
+import UserDisconnectLoginAccountMutation from '../../relay/mutations/UserDisconnectLoginAccountMutation';
+import { login } from '../../redux/actions';
+import {
+  FlexRow,
+  Text,
+  slackGreen,
+  twitterBlue,
+  facebookBlue,
+} from '../../styles/js/shared';
+
+const messages = defineMessages({
+  disconnectAccount: {
+    id: 'UserConnectedAccount.disconnectAccount',
+    defaultMessage: 'Disconnect Account',
+  },
+  typeHere: {
+    id: 'UserConnectedAccount.typeHere',
+    defaultMessage: 'Type here',
+  },
+  confirmError: {
+    id: 'UserConnectedAccount.confirmError',
+    defaultMessage: 'You should type "confirm"',
```

Comment:
@infojunkie shouldn't we avoid asking users to type `confirm`?

Reply:
IIRC we have settled for a checkbox-based confirmation pattern. For this purpose I have created a reusable `ConfirmDialog` component. It's currently being used in Team Tasks. See `TeamTasksListItem`

Reply:
Done and used `ConfirmDialog`